src/scipp/core/like.py

```diff
@@ -23,15 +23,26 @@ def zeros_like(
     var: _Union[_cpp.Variable,
                 _cpp.DataArray]) -> _Union[_cpp.Variable, _cpp.DataArray]:
     """
-    Constructs a new object with the same dims, shape, unit and dtype as the input
-    (:class:`Variable` or :class:`DataArray`), but with all values initialized to 0.
+    Return a Variable or DataArray with the same dims, shape, unit, and dtype as the
+    input and all values initialized to 0.
+
     If the input has variances, all variances in the output are set to 0.
-    If the input is a :class:`DataArray`, coordinates and attributes are shallow-copied
+    If the input is a data array, coordinates and attributes are shallow-copied
```

Comment:
The help strings are definitely improved.

Reply:
Agree. But why didn't you use `DataArray` here? If I understand correctly, that would produce a link in sphinx.

Reply:
That would link to the class, I don't know if we want to do that everywhere, even when referring to an instance? Note that we could define a *term* "data array", and it would produce a link to a glossary.

Reply:
I personally find it slightly confusing that there are 2 names for DataArrays. The 'technical' DataArray and the 'colloquial' data array. I think we should stick to the technical one because 'data array' could also be a more general term.

Reply:
We don't in notebooks because it is a pain to type and hurts readability, I think. But we can change it in docstrings?

Reply:
This comes back to maybe finding a better name for `DataArray`? but i still don't have a good idea

Reply:
Not really? In my view this is simply a question of whether we want to distinguish classes vs instances of class in user-facing documentation, and if so, how to refer to them. The Jupyter notebooks usually just use the names variable and data array to refer to instances, since having links every may hurt readability for longer doc pages. For docstrings this is slightly different: Having links there may actually be useful, but we could link from the lower case terms ("variable" and "data array"), to make typing and reading the non-Sphinx docstrings easier.